lib/random_assign_utils.rb

```diff
@@ -38,19 +38,23 @@ def self.automation_script!(context, fields, automation)
       raise_error(automation, "Group(#{group_id}) not found")
     end
 
+    allowed_assign_user_ids = User.assign_allowed.pluck(:id)
```

Comment:
Maybe a better name would be?

```suggestion
    assignable_user_ids = User.assign_allowed.pluck(:id)
```